src/views/Hearing/index.js

```diff
@@ -120,7 +120,8 @@ class Hearing extends React.Component {
     const hearingAllowsComments = !hearing.closed && (new Date() < new Date(hearing.close_at));
     const onPostVote = this.onVoteComment.bind(this);
     const introSection = detect(hearing.sections, (section) => section.type === "introduction");
-    const regularSections = hearing.sections.filter((section) => section.type !== "introduction");
+    const closureInfoSection = detect(hearing.sections, (section) => section.type === "closure info");
+    const regularSections = hearing.sections.filter((section) => !["introduction", "closure info"].includes(section.type));
```

Comment:
This could now be refactored to use [_.difference](https://lodash.com/docs#difference):

```js
const regularSections = difference(hearing.sections, [introSection, closureInfoSection]);
```

:)